root/layout/components/sidebar/SidebarLicenses.js

```diff
@@ -20,23 +20,59 @@ import {l} from '../../../static/scripts/common/i18n';
 import {l_relationships} from '../../../static/scripts/common/i18n/relationships';
 
 const LICENSE_CLASSES = {
-  ArtLibre: /artlibre\.org\/licence\/lal/,
-  CC0: /creativecommons\.org\/publicdomain\/zero\//,
-  CCBY: /creativecommons\.org\/licenses\/by\//,
-  CCBYNC: /creativecommons\.org\/licenses\/by-nc\//,
-  CCBYNCND: /creativecommons\.org\/licenses\/by-nc-nd\//,
-  CCBYNCSA: /creativecommons\.org\/licenses\/by-nc-sa\//,
-  CCBYND: /creativecommons\.org\/licenses\/by-nd\//,
-  CCBYSA: /creativecommons\.org\/licenses\/by-sa\//,
-  CCNCSamplingPlus: /creativecommons\.org\/licenses\/nc-sampling\+\//,
-  CCPD: /creativecommons\.org\/licenses\/publicdomain\//,
-  CCSampling: /creativecommons\.org\/licenses\/sampling\//,
-  CCSamplingPlus: /creativecommons\.org\/licenses\/sampling\+\//,
+  ArtLibre: {
+    icon: manifest.pathTo('/images/licenses/ArtLibre.png'),
```

Comment:
There's a way to do dynamic imports like ``require(`../images/licenses/${className}.png`)`` with Webpack, though I haven't tested it and it's easier to just specify them all separately.